src/playsim/p_map.cpp

```diff
@@ -492,6 +492,9 @@ bool	P_TeleportMove(AActor* thing, const DVector3 &pos, bool telefrag, bool modi
 		if (tmf.thing->flags6 & MF6_THRUSPECIES && tmf.thing->GetSpecies() == th->GetSpecies())
 			continue;
 
+		if ((multiplayer && !deathmatch) && (dmflags3 & DF3_NO_PLAYER_CLIP) && (tmf.thing->player && th->player))
```

Comment:
Same as my other comment

Reply:
@madame-rachelle this is very much necessary, otherwise players will telefrag each other. The whole motivation of sv_noplayerclip is to allow stress-free coop games where players absolutely cannot hurt or block each other.